src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -90,15 +90,21 @@ public static List<PartyDetails> buildHearingPartiesDetails(SscsCaseData caseDat
                 buildHearingPartiesPartyDetails(
                     caseData.getJointParty(),
                     appellant.getId(),
-                    referenceDataServiceHolder
+                    referenceDataServiceHolder,
+                    null
                 ));
         }
 
         OverrideFields overrideFields = OverridesMapping.getOverrideFields(caseData);
 
+        String adjournLanguage = null;
```

Comment:
Can you replace the variable name adjournLanguage (and in the following params) with something like adjournLanguageRef, so that is clear it is a hmcReference not the name of a language name being passed